### PR TITLE
feat: WDS colors `bgAccentSubtle*` (light mode)

### LINKED ISSUE #24034: bgAccentSubtle (light mode)

### PR DESCRIPTION
**Pull request by ichik**
## Description

tl;dr Better logic and looks for light mode `bgAccentSubtle*` state colors 

Fixes #24034  

## Type of change
- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Chore (housekeeping or task changes that don't impact user perception)

## How Has This Been Tested?

- Manual

### Test Plan
Initial POC refinement, no testing necessary

### Dev activity
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

### QA activity:
- [ ] Test plan has been approved by relevant developers
- [ ] Test plan has been peer reviewed by QA
- [ ] Cypress test cases have been added and approved by either SDET or manual QA
- [ ] Organized project review call with relevant stakeholders after Round 1/2 of QA
- [ ] Added Test Plan Approved label after reveiwing all Cypress test

### CODE REVIEW

File: app/client/packages/design-system/theming/src/color/LightModeTheme.ts [hunk 1]
```diff
@@ -239,11 +239,19 @@ export class LightModeTheme implements ColorModeTheme {
   }
 
   private get bgAccentSubtleHover() {
-    return this.bgAccentSubtle.clone().lighten(0.02);
+    const color = this.bgAccentSubtle.clone();
+
+    color.oklch.l = color.oklch.l + 0.02;
```
Comment (KelvinOm) [inline app/client/packages/design-system/theming/src/color/LightModeTheme.ts:244]: (nitpick) You can write it shorter `color.oklch.l += 0.02;` 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Addition_assignment

**Review by KelvinOm — APPROVED** [review]
